SwiftWisdom/Core/UIKit/UIView+SafeArea.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  UIView+SafeArea.swift
+//  SwiftWisdom
+//
+//  Created by Persian, Alex on 11/3/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+extension UIView {
+
+    /// Provides the view's edge insets with regards to the safe area.
+    ///
+    /// **11.0 and above**, returns `safeAreaInsets`
+    ///
+    /// **10.x and below**, returns `UIEdgeInsets.zero`
+    var ip_safeAreaInsets: UIEdgeInsets {
+        if #available(iOS 11.0, *) {
+            return self.safeAreaInsets
+        } else {
+            return UIEdgeInsets.zero
+        }
+    }
+
+    /// Provides the view's top anchor with regards to the safe area.
+    ///
+    /// **Note:** for iOS 11 and above this anchor will not be available
+    /// until after `viewDidAppear` is called
```

Comment:
nit: might want to clarify that it is _available_ before `viewDidAppear`, but it is not *accurate* until it has been added to the view hierarchy and `viewDidAppear` is called